README.md

```diff
@@ -43,7 +43,7 @@ In addition to the AWS Solutions Constructs, the solution uses AWS CDK directly
 
 # Customizing the Solution
 
-## Prerequisites for Customization
+### Prerequisites for Customization
```

Comment:
A second-level heading seemed right here, should this really be at the same level as the steps?

Reply:
addressed